samza-core/src/test/java/org/apache/samza/container/grouper/task/TestGroupByContainerCount.java

```diff
@@ -320,7 +328,7 @@ public void testBalancerAfterContainerDecrease() {
    *  T8  T7  T3
    */
   @Test
-  public void testBalancerMultipleReblances() {
+  public void testBalancerMultipleReblances() throws Exception {
```

Comment:
"Rebalances". Also, why does the test `throws Exception`?

Reply:
Mocking via `PowerMock` bubbles up an exception, hence there was a `throws` clause.  I will remove it in my follow-up patch(part of SEP-11).

Reply:
Please don't resolve comments right after replying. This collapses them in the UI and makes it difficult to follow up.